src/features/PeopleList/PersonTile/styled.js

```diff
@@ -0,0 +1,57 @@
+import styled from "styled-components";
+
+export const Content = styled.div`
+    width: 208px;
```

Comment:
There is no point in setting width here, while it is already set in grid-template-columns. Also why it is 208px here and 200px in grid?